pkg/config/analysis/analyzers/util/in_mesh.go

```diff
@@ -57,11 +66,11 @@ func inMesh(annos map[string]string, namespace resource.Namespace, containers []
 }
 
 // getPodSidecarInjectionStatus returns two booleans: enabled and ok.
-// enabled is true when deployment d PodSpec has either the annotation 'sidecar.istio.io/inject: "true"'
-// ok is true when the PodSpec doesn't have the 'sidecar.istio.io/inject' annotation present.
+// enabled is true when deployment d PodSpec has either the label/annotation 'sidecar.istio.io/inject: "true"'
+// ok is true when the PodSpec doesn't have the 'sidecar.istio.io/inject' label/annotation present.
 func getPodSidecarInjectionStatus(annos map[string]string) (enabled bool, ok bool) {
-	v, ok := annos[annotation.SidecarInject.Name]
-	return v == "true", ok
+	v, ok := annos[label.SidecarInject.Name]
+	return strings.EqualFold(v, "true"), ok
```

Comment:
`TRUE` is not valid

Reply:
I am not sure, actually in some other places we compare with `EqualFold`. 

Can you confirm?

Reply:
Also ParseBool is used